pkg/controller/releasemanager/syncer.go

```diff
@@ -148,8 +148,12 @@ func (r *ResourceSyncer) reportMetrics() error {
 		current := incarnation.status.State.Current
 		incarnationsInState[current]++
 
-		lastUpdated := incarnation.status.LastUpdated.Time
-		age := time.Since(lastUpdated).Seconds()
+		var lastUpdated *time.Time
+		var age float64
+		if incarnation.status.State.LastUpdated != nil {
```

Comment:
what is the difference between status.LastUpdated and status.State.LastUpdated?

Reply:
I might do another PR where I remove most of the crud in status. state lastUpdated is the last time the state changed, which is what we should use for timeouts. I'm not sure if status lastUpdated is used anywhere. It should be at least as up-to-date as state lastUpdated, but it's not, so there's that..